src/input/target/horipad_steam.rs

```diff
@@ -318,6 +318,19 @@ impl TargetInputDevice for HoripadSteamDevice {
         let _ = self.device.destroy();
         Ok(())
     }
+
+    /// Clear any local state on the target device.
+    fn clear_state(&mut self) {
+        let caps = self.get_capabilities().unwrap_or_else(|_| {
+            log::error!("No target device capabilities found while clearing state.");
+            Vec::new()
+        });
+        for cap in caps {
+            let ev = NativeEvent::new(cap, InputValue::Bool(false));
+            self.queued_events
+                .push(ScheduledNativeEvent::new(ev, Duration::from_millis(0)));
+        }
+    }
```

Comment:
Like the DualSense, any target devices that are hidraw or usb based can simply reset the gamepad state instead of queuing events:

```suggestion
    /// Clear any local state on the target device.
    fn clear_state(&mut self) {
        self.state = Default::default();
    }
```